tests_integ/helpers/helpers.py

```diff
@@ -0,0 +1,91 @@
+import json
+import logging
+import re
+import random
+import string  # not deprecated, a bug from pylint https://www.logilab.org/ticket/2481
+from functools import reduce
+
+import boto3
+from botocore.exceptions import ClientError, NoRegionError
+
+from samtranslator.model.exceptions import InvalidDocumentException
+from samtranslator.translator.managed_policy_translator import ManagedPolicyLoader
+from samtranslator.translator.transform import transform
+from samtranslator.yaml_helper import yaml_parse
+
+RANDOM_SUFFIX_LENGTH = 12
+
+
+def transform_template(input_file_path, output_file_path):
+    LOG = logging.getLogger(__name__)
+    iam_client = boto3.client("iam")
+
+    with open(input_file_path, "r") as f:
+        sam_template = yaml_parse(f)
+
+    try:
+        cloud_formation_template = transform(sam_template, {}, ManagedPolicyLoader(iam_client))
+        cloud_formation_template_prettified = json.dumps(cloud_formation_template, indent=2)
+
+        with open(output_file_path, "w") as f:
+            f.write(cloud_formation_template_prettified)
+
+        print("Wrote transformed CloudFormation template to: " + output_file_path)
+    except InvalidDocumentException as e:
+        error_message = reduce(lambda message, error: message + " " + error.message, e.causes, e.message)
+        LOG.error(error_message)
+        errors = map(lambda cause: cause.message, e.causes)
+        LOG.error(errors)
+
+
+def verify_stack_resources(expected_file_path, stack_resources):
+    with open(expected_file_path, "r") as expected_data:
+        expected_resources = _sort_resources(json.load(expected_data))
+    parsed_resources = _sort_resources(stack_resources["StackResourceSummaries"])
+
+    if len(expected_resources) != len(parsed_resources):
+        return False
+
+    for i in range(len(expected_resources)):
+        exp = expected_resources[i]
+        parsed = parsed_resources[i]
+        if not re.fullmatch(exp["LogicalResourceId"] + "([0-9a-f]{10})?", parsed["LogicalResourceId"]):
```

Comment:
Why `{10}`?

Reply:
That's the length of the random string being added to logical IDs for some resources (not all) once deployed 
I moved it to a constant